apache-rat/src/site/apt/index.apt.vm

```diff
@@ -75,17 +75,17 @@ java -jar apache-rat/target/apache-rat-${project.version}.jar
 
 
 +------------------------------------------+
-usage: java -jar apache-rat/target/apache-rat-${project.version}.jar
- [options] [DIR|TARBALL]
+usage: java -jar apache-rat/target/apache-rat-CURRENT-VERSION.jar
```

Comment:
I've asked the same question in the other PR - could we stick with project.version here in order to generate the correct version into the file?

Reply:
Fixed

Reply:
FYI, I've updated site/README with all missing options (copy/paste of the `--help` output).